Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSBeamCentreFinder.py

```diff
@@ -23,12 +23,18 @@
 from mantid import AnalysisDataService
 from mantid.simpleapi import CloneWorkspace, GroupWorkspaces
 
-try:
-    import mantidplot
-except (Exception, Warning):
-    mantidplot = None
-# this should happen when this is called from outside Mantidplot and only then,
-# the result is that attempting to plot will raise an exception
+from qtpy import PYQT4
+if PYQT4:
```

Comment:
Is it worth setting `IN_MANTIDPLOT = False` by default just above this line and only use `IN_MANTIDPLOT` check later on so you don't need to check for both Qt4 and `MANTIDPLOT` all of the place?

Reply:
I don't think it will make a change to the checks called in SANSBeamCentreFinder, but I've updated the other files where we use this check